dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/task/StatsJdbcTask.java

```diff
@@ -69,6 +70,17 @@ public StatsJdbcTask onlyIfFailed(Task<?> prerequisite) {
     return new StatsJdbcTask(query, failureCondition);
   }
 
+  @Nonnull
+  public static StatsJdbcTask onlyIfAllSkipped(
```

Comment:
When I look at the implementation without looking into more details, I would say that the method should be called `onlyIfAllSucceeded`. Why is it called `onlyIfAllSkipped`? How does it work?

Reply:
`SUCCEEDED` was incorrect here, it should have been `SKIPPED`. I've fixed the condition, thanks for catching this